### ISSUE TITLE
[BUG] Sending ctrl-alt-delete does not have any reaction on a windows VM

### ISSUE THREAD

**Issue by Martin-Weiss** [harvester/harvester#4535]
**Describe the bug**
Here we do not have any reaction:

![grafik](https://github.com/harvester/harvester/assets/13362573/3154e6fe-9f42-4cc7-a503-87b8af1a62ec)

**To Reproduce**
Have a windows 2022 VM and try to send ctrl-alt-del

**Expected behavior**
ctrl-alt-del is received and we can login

**Environment**
 - Harvester ISO version: 1.2.0
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): not relevant
 - Browser: Firefox 117.0 on Mac (M1)

**Additional context**
Workaround on Mac: use control+option+delete on the keyboard

**Comment by joshfranzen** [comment]
Same issue here running on Harvester 1.2.0

here is the VM config:

```
spec:
  runStrategy: RerunOnFailure
  template:
    metadata:
      annotations:
        harvesterhci.io/sshNames: '[]'
        harvesterhci.io/waitForLeaseInterfaceNames: '[]'
      creationTimestamp: null
      labels:
        harvesterhci.io/creator: terraform-provider-harvester
        harvesterhci.io/vmName: d-n-win-image
    spec:
      affinity:
        podAntiAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
            - podAffinityTerm:
                labelSelector:
                  matchExpressions:
                    - key: harvesterhci.io/creator
                      operator: Exists
                topologyKey: kubernetes.io/hostname
              weight: 100
      domain:
        cpu:
          cores: 4
        devices:
          disks:
            - name: cdrom-disk
              cdrom:
                bus: sata
              bootOrder: 1
            - name: osdisk
              disk:
                bus: sata
              bootOrder: 2
          inputs:
            - bus: usb
              name: tablet
              type: tablet
          interfaces:
            - masquerade: {}
              macAddress: 52:54:00:e1:5c:be
              model: e1000
              name: nic-1
        features:
          acpi: {}
          smm:
            enabled: true
        firmware:
          bootloader:
            efi:
              secureBoot: true
        machine:
          type: q35
        memory:
          guest: 1948Mi
        resources:
          limits:
            cpu: '4'
            memory: 2Gi
          requests:
            cpu: 250m
            memory: 1365Mi
      evictionStrategy: LiveMigrate
      hostname: d-n-win-image
      networks:
        - name: nic-1
          pod: {}
      terminationGracePeriodSeconds: 120
      volumes:
        - name: cdrom-disk
          persistentVolumeClaim:
            claimName: d-n-win-image-cdrom-disk-cpfhh
        - name: osdisk
          persistentVolumeClaim:
            claimName: d-n-win-image-osdisk-bphjg
```

**Comment by alexey-rawyan** [comment]
as for now you can create management shortcut key (record new one with ctrl then alt then del, and save it) - it works.
![image](https://github.com/harvester/harvester/assets/134208995/fe965fad-c423-4a09-9e0d-bd5107f9c601)

**Comment by W0n9** [comment]
The same problem here, and custom shortcut works.

**Comment by gaktive** [comment]
@bk201 I see the PR was merged back on that day but this ticket was reopened. Is there something left to do here? 

Internal reference: SURE-7210

**Comment by bk201** [comment]
@torchiaf Can you update the test plan in https://github.com/harvester/harvester/issues/4535#issuecomment-1875001377 and move the issue to ready for testing? And we also need to backport this to v1.2.2.

**Comment by bk201** [comment]
> @bk201 I see the PR was merged back on that day but this ticket was reopened. Is there something left to do here?
> 
> Internal reference: SURE-7210

@gaktive The issue was reopened because the issue was accidentally closed by keywords in the issue's PR. The internal ticket will be marked as resolved once QA verifies the issue.

**Comment by TachunLin** [comment]
Verified **fixed** on `v1.3.0-rc1` (01/26). Close this issue.

### Result
<details><summary>$\color{green}{\textsf{PASS}}$ Click ctrl+alt+delete key combination works on windows login page $~~$</summary>

* Click the ctrl+alt+delete key combination on the windows server 2022 login page works as expected

https://github.com/harvester/harvester/assets/29251855/3260dd1c-748e-4633-8084-54180660489d


 </details>

<details><summary>$\color{green}{\textsf{PASS}}$ Click ctrl+alt+delete key combination works inside windows  $~~$</summary>

* Click the ctrl+alt+delete key combination after login to windows server 2022 desktop works as expected


https://github.com/harvester/harvester/assets/29251855/a9d86199-75ca-46ce-9949-fbee7d3a20e6


 </details>

### Test Information
* Test Environment: Single nodes harvester on local kvm machine
* Harvester version: `v1.3.0-rc1` (01/26)
* Windows ISO: windows_server_2022_en-us

### Verify Steps
<details><summary> On Windows server 2022 </summary>

  1. Create windows server 2022 iso image
  2. Create a VM, add the second `SATA` disk
  3. Install windows server 2002 with desktop 
  4. After installation complete
  5. Open the web console 
  5. In the windows login page, click the Shortcut Keys
  6. Click Ctrl , Alt, Del in sequence
  7. After login to desktop
  8.   Click Ctrl , Alt, Del in sequence again

</details>
